universal-application-tool-0.0.1/test/services/role/RoleServiceTest.java

```diff
@@ -53,49 +53,6 @@ public void makeProgramAdmins_allPromoted() throws ProgramNotFoundException {
     assertThat(account2.getAdministeredProgramNames()).containsOnly(programName);
   }
 
-  @Test
```

Comment:
Can we add a test that verifies that both of the emails are added as program admins?

Reply:
Not sure I understand what case that is covering. Can you explain what the purpose of that test would be?

If we add both emails as program admins and then look them up separately, we're not really proving anything -- that test would probably also pass for case-insensitive logic. Are you suggesting that I add a test to check case sensitivity? If so, we should probably add a test that saves an account with an upper case email, and then looks up the account with the lower case email, and expect the found account to not have any programs. 

What do you think?

Reply:
Added a test to check the lower case and upper case email admin permissions!